### ISSUE TITLE
Normalization on input depth and flow?

### ISSUE THREAD

**Issue by Shuang1997** [Brummi/anycam#9]
Hi, thanks for releasing your great work!

I have went through the code. It seems that you normalize the input RGB images, but I don't see any normalization on the input depth and flow data. Would this cause a problem? I am thinking of this because you directly concat rgb, flow and depth as the input to DINOv2, which is only pretrained on (normalized) RGB images. 

Looking forward to seeing your opinion!

**Comment by Brummi** [comment]
Hi!

thank you for your interest in our work!

We indeed do not normalize depth or flow, because for both of them, the network needs to know the true scale. E.g. the magnitude of the translation is dependent on the depth values.

The lack of normalisation is no problem since we fine-tune the DinoV2 backbone during training.

Hope this helps!

Best,
Felix

**Comment by Shuang1997** [comment]
Good to know! Thanks for your quick response!
